var/spack/repos/builtin/packages/id3lib/id3lib-3.8.3_gcc4.diff

```diff
@@ -0,0 +1,102 @@
+Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
```

Comment:
Wow... It looks like the ID3LIB source code hasn't changed since 2003, and so this patch is still necessary!